### DISCUSSION TITLE
i do not know default account

### DISCUSSION THREAD

**Discussion by a137872798** [kanidm/kanidm#1808] [General]
when i install kanidm ui, i can not login because of i do not know account.

**Comment by Firstyear** [comment]
This is covered in https://kanidm.github.io/kanidm/stable/accounts_and_groups.html

**Reply by a137872798** [reply]
thanks for answer, when i use admin to login. 
![image](https://github.com/kanidm/kanidm/assets/35627706/75a527e2-c51d-426e-80db-0f33d9747114)
return the message, but i have reset password

**Reply by Firstyear** [reply]
Admin is a service account not a person. 
Admin recovers idm_admin.
idm_admin creates a person
login with the person
:)

**Comment by a137872798** [comment]
i run kanidm in docker, and install kanidm client tool in host machine. when i use kanidm to create person account. client tool tell me only support command
![image](https://github.com/kanidm/kanidm/assets/35627706/2bd918d6-795c-47c2-b3fe-5a14f969af99)
i can not use 
![image](https://github.com/kanidm/kanidm/assets/35627706/c689c3b0-b4e6-449d-af55-7212063b3f08)

**Reply by Firstyear** [reply]
This is documented here: https://kanidm.github.io/kanidm/stable/installing_client_tools.html 

The book is laid out to allow you to follow it from start to finish and get a working system. You likely need to configure the client configuration here.

**Reply by Firstyear** [reply]
Also your client tools are too old I suspect. Install them from the above documentation.

**Reply by a137872798** [reply]
it is right. now i use cargo search find latest version. thank you.
